app/controllers/application_controller.rb

```diff
@@ -99,12 +99,12 @@ def find_user
 
   def find_asset
     @asset = @user.assets.where(permalink: (params[:permalink] || params[:track_id] || params[:id])).first
-    @asset ||= @user.assets.where(id: params[:id]).first || track_not_found
+    @asset ||= @user.assets.where(id: params[:id]).first || track_not_found_message
```

Comment:
This change and the other in the file also probably need reversion — we need to raise here, not output a string :D